app/payment-link-v2/reference/get-index.controller.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const { response } = require('../../utils/response')
+const { getSessionVariable } = require('../../utils/cookie')
+const { NotFoundError } = require('../../errors')
+const getBackLinkUrl = require('./get-back-link-url')
+
+module.exports = (req, res, next) => {
```

Comment:
We shouldn't export anonymous functions as per https://pay-team-manual.cloudapps.digital/manual/coding-standards/node-js-style-guide.html#anonymous-functions

Reply:
Thanks - have updated this.